src/article-list-view.js

```diff
@@ -0,0 +1,42 @@
+(function(exports){
+  function ArticleListView(articleList = new ArticleList){
+    this._articleList = articleList
+    this._list = articleList.list();
+    this._result = '';
+  }
+  function renderToHtml(article){
+    var content = imgHtml(article)+titleHtml(article)+summaryHtml(article)
+    return `<div id="${article.id()}">${content}</div>`
+  }
+
+  function titleHtml(article){
+    return `<div class="title"><a href="#article/${article.id()}">${article.title()}</a></div>`
+  }
+  function imgHtml(article){
+    return `<div class="image"><a href="#article/${article.id()}">${article.img()}</a></div>`
+  }
+
+  function summaryHtml(article){
+    return `<div class="summary">${article.summary()}</div>`
+  }
+
+  ArticleListView.prototype.convert = function(){
+    var self = this;
+    this._list.forEach(function(article){self._result+= renderToHtml(article)})
+
+  }
+
+  ArticleListView.prototype.toHtml = function(){
+    return this._result;
+  }
+
+  ArticleListView.prototype.addArticle = function(data, body){
+    this._articleList.add(data, body);
```

Comment:
It feels odd that you need to pass an ArticleModel into the view in order to add it to the ArticleListModel - this feels like tight coupling. I would expect you to be able to add the ArticleModel directly to the ArticleViewModel and the for the ArticleListView not to need this method or to know anything about how the ArticleListModel is built.